templates/zerver/integrations/openshift.md

```diff
@@ -0,0 +1,54 @@
+Ths integration sends a notification every time a deployment is made in an OpenShift instance.
+
+First, download our [Pythonbindings and example scripts](/api), and move it to your local copy of the application’s
+repository.
+
+Then, create a new commit including all the changes made to the repository, and push it to your app.
+
+After that, connect to the application through SSH. If you don’t know how to do this, log in to your OpenShift
+Online account, go to your application’s dashboard, and click “Want to log in to your application?”.
+There you’ll find the app’s SSH user, address, and further information on SSH, in case you need it.
+
+![](/static/images/integrations/openshift/002.png)
+
+Once you have connected, install the Python bindings by running:
+
+```bash
+$ cd ~/app-root/repo/<python_bindings_path>
+$ python setup.py install
+```
+
+Where `<python_bindings_path>` is the path where you saved the bindings in the first step.
+
+Next, open `integrations/openshift/zulip_openshift_config.py` inside the SSH terminal with your favorite editor, and
+change the following lines to specify the email address and API key for your OpenShift integration:
+
+```
+ZULIP_USER = "openshift-bot@example.com"
+ZULIP_API_KEY = "0123456789abcdef0123456789abcdef"
+{% if api_site_required %}ZULIP_SITE = "{{ external_api_uri_subdomain }}"{% endif %}
+```
+
```

Comment:
I feel like someone mentioned this on the Dropbox Paper instance documenting our migration efforts. We might want to create a macro for lines 23-30 (or just lines 27-29). It seems like we are using similar code blocks in a lot of these files. :)